src/components/admin/UserProfileModal/components/links/ProfileModalEditLink.tsx

```diff
@@ -42,34 +40,41 @@ export const ProfileModalEditLink: React.FC<ProfileModalEditLinkProps> = ({
   );
 
   return (
-    <div className={classNames("ProfileModalEditLink", containerClassName)}>
-      <div className="ProfileModalEditLink__url">
-        <ProfileModalInput
-          placeholder="Link URL"
-          name={`profileLinks.${index}.url`}
-          register={register}
-          rules={{
-            required: "URL cannot be empty",
-            pattern: {
-              value: urlRegex,
-              message: "URL must be valid",
-            },
-            validate: validateURLUnique,
-          }}
-          error={error?.url}
-        />
-      </div>
-      <div className="ProfileModalEditLink__text">
-        <ProfileModalInput
-          placeholder="Link Title"
-          register={register}
-          name={`profileLinks.${index}.title`}
-          rules={{
-            required: "Title cannot empty",
-          }}
-          error={error?.title}
-        />
+    <div
+      className={classNames("ProfileModalEditLink flex", containerClassName)}
```

Comment:
Same here for the `.scss` file classNames.

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8